styles.css

```diff
@@ -89,6 +104,171 @@ header {
   height: 100%;
 }
 
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 24px;
+}
+
+.works-title-section {
+  margin-bottom: 80px;
+  text-align: center;
+}
+
+.works-title {
+  font-size: 40px;
+  line-height: 52px;
+  height: 83px;
+  margin-top: 120px;
+  margin-bottom: 24px;
+}
+
+.f-project-img {
+  width: 100%;
+  height: 237px;
+}
+
+.project-title {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: normal;
+}
+
+.project-description {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.project {
+  display: flex;
+  align-items: flex-end;
+  width: 100%;
+  height: 386px;
+  background: #aaa;
+  background-image: url("./assets/png/art\ printing.png");
+  background-position: top left;
+  background-size: cover;
+  padding: 16px;
+}
+
+.about {
+  margin-top: 120px;
+  background: #1c1a19;
+  padding-bottom: 80px;
+}
+
+.about-intro {
+  width: 100%;
+  padding-top: 120px;
+  padding-bottom: 209px;
+  background-image: url("assets/png/illustration-about-me-mobile.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+}
+
+.about-title {
+  height: 79px;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.33px;
+}
+
+.about-separater {
+  border: 1px solid #fff;
+  margin-bottom: 136px;
+}
+
+.technologies {
+  list-style: none;
+  width: 100%;
+  text-align: center;
+}
+
+.tech-title {
+  height: 41.46px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 12px;
+  margin-top: 2px;
+  margin-bottom: 2px;
+}
+
+.tech-title > h5 {
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.tech {
+  font-size: 22px;
+  line-height: 43px;
+  list-style: none;
+  text-align: center;
+  height: 48px;
+}
+
+.tags {
+  display: flex;
+  gap: 12px;
+  justify-content: flex-start;
+}
+
+.tag {
+  border: 1px solid #8993a4;
+  padding: 10px;
+  font-size: 15px;
+  line-height: 20px;
+  list-style: none;
+}
+
+.dark-tag {
+  background: #fff;
+  opacity: 0.24;
+}
+
+.button {
+  padding: 12px;
+  font-family: Inter, sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  background: #ff6b00;
+  color: #fff;
+}
+
+.button:hover {
+  transition: all 0.2s ease-in-out;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
```

Comment:
[REQUIRED] While you have done a great job implementing the style of the buttons, you missed the style when the buttons are pressed.

- [ ] Please make sure to implement the style for the buttons when active accordingly with the following figma design:
![image](https://user-images.githubusercontent.com/83596135/153658431-6060de0f-b6e5-4f6a-97eb-03cf013d096c.png)

Reply:
Hi @nicupop729 thanks for the review, I'm done with the required changes.